ethcore/trace/src/types/flat.rs

```diff
@@ -96,7 +97,7 @@ impl Into<Vec<FlatTrace>> for FlatTransactionTraces {
 
 /// Represents all traces produced by transactions in a single block.
 #[derive(Debug, PartialEq, Clone, Default, RlpEncodableWrapper, RlpDecodableWrapper, MallocSizeOf)]
-pub struct FlatBlockTraces(Vec<FlatTransactionTraces>);
+pub struct FlatBlockTraces(pub(crate) Vec<FlatTransactionTraces>);
```

Comment:
why is it `pub(crate)` now?

Reply:
I honestly don't remember. It was necessary at some point during the refactor but afaict it works fine now without it. Fixing.

Reply:
I found it: it's a new error (i.e. a warning is now promoted to error):

```
error: private struct constructors are not usable through re-exports in outer modules
   --> ethcore/trace/src/types/flat.rs:226:4
    |
226 |             FlatTransactionTraces(vec![flat_trace]),
    |             ^^^^^^^^^^^^^^^^^^^^^
    |
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #39207 <https://github.com/rust-lang/rust/issues/39207>
```

Reply:
Interestingly that error shows up only if I run `cargo test -p trace` and not with `cargo test --all`. Hmm.

Reply:
@dvdplm Perhaps new crates need to be added to the workspace? Can you double check that the tests are actually run on the CI?

Reply:
@tomusdrw The failures do show up in CI (but not locally with `--all`). :/